source/systems/gaea_user_guide.rst

```diff
@@ -91,2113 +93,1545 @@ points at Atlanta and Chicago.
+taking longer than normal to complete, or timming out. Many jobs on Gaea are
+currently experiencing longer than normal run times.  While we do not yet have
+an underlying cause for this, we have found certain changes to the user's
+interactions and workflows that use the GPFS F5 file system help alleviate the
+problem.
 
-To change the default loaded PGI environment to the default version of
-GNU use:
+Files accesses by multiple jobs
+-------------------------------
 
-.. code-block:: shell
+Users should not have multiple batch jobs access the same files.  This is
+typically done using hard- or soft-links.  Access the same file from multiple
+batch jobs increases the load on the metadata servers (MDS), and can lead to a
+MDS locking up affecting all files served on that MDS.
 
-  $ module unload PrgEnv-pgi $ module load PrgEnv-gnu
+Users should clean up files after the job runs successfully to ensure the file
+system has enough free space for all user's jobs.
 
-**Changing Versions of the Same Compiler**
+Software Environments
+---------------------
 
-To use a specific compiler version, you must first ensure the
-compiler's PrgEnv module is loaded, and then swap to the correct
-compiler version. For example, the following will configure the
-environment to use the GCC compilers, then load a non-default GCC
-compiler version:
+Users should not store software environments, e.g., conda, spack, on the GPFS
+file system.  These environments have many small files that will be accessed
+from multiple compute nodes when used in batch jobs.
 
-.. code-block:: shell
+These environments should be stored in user's or project's home space,
+:file:`/ncrc/home[12]/$USER` and :file:`/ncrc/proj/<project>` respectively.  If
+the environment is to be shared by several users or groups, the environment can
+in the :file:`/usw`.  Please open a :ref:`help desk request <getting_help>` to
+establish a location under :file:`/usw`.
 
-  $ module swap PrgEnv-pgi PrgEnv-gnu
-  $ module swap gcc gcc/4.6.2
+Development
+-----------
 
-**General Programming Environment Guidelines**
+GPFS F5 should not be used for development.  Development should be done in the
+user's home space.  This is especially true if using a source code management
+system (e.g., git).
 
-We recommend the following general guidelines for using the
-programming environment modules:
+Users should remember that GPFS F5 is not backed up. The user home area is
+backed up, with hourly and daily snapshots.
 
-- Do not purge all modules; rather, use the default module environment
-  provided at the time of login, and modify it.
-- Do not swap or unload any of the Cray provided modules (those with
-  names like xt-'*').
-- Do not swap moab, torque, or MySQL modules after loading a
-  programming environment modulefile.
+************
+Known issues
+************
 
-Compiling Threaded Codes
-------------------------
+These are a list of known issues we are currently investigating on the Gaea
```

Comment:
The following is a list of issues we are currently investigating.
Also, the <getting_help>` link isn't right -- ref missing? 

:ref:`help desk request <getting_help>`

Reply:
Adjusted in new push.